packages/core/src/external/athenahealth/index.ts

```diff
@@ -337,6 +339,14 @@ class AthenaHealthApi {
   private stripDepartmentId(id: string) {
     return id.replace(`a-${this.practiceId}.${athenaDepartmentPrefix}-`, "");
   }
+
+  private formatDate(date: string | undefined): string | undefined {
+    if (!date) return undefined;
+    const trimmedDate = date.trim();
+    const parsedDate = dayjs(trimmedDate);
```

Comment:
nit: we started to use `buildDayjs()` to build those using `dayjs.utc()`, so dates are created as UTC, regardless of the TZ we have on our local env.

Reply:
Done.